src/components/pages/lessons/LessonInfo.tsx

```diff
@@ -318,9 +320,9 @@ const LessonInfo: FunctionComponent<LessonInfoProps> = ({
           </div>
         </div>
       </div>
-      {nextUpData && (
+      {!playerState.matches('loading') && nextUpUrl && (
```

Comment:
if the player is loading, hold off on the next up list. Hoping to provide the player with priority.